autorest/adal/token.go

```diff
@@ -54,6 +54,9 @@ const (
 
 	// metadataHeader is the header required by MSI extension
 	metadataHeader = "Metadata"
+
+	// imdsEndpoint is the well known endpoint for getting authentications tokens inside Azure virtual machines
```

Comment:
I would change the comment to say that IDMS has a well known endpoint for obtaining an MSI token

Reply:
Fixed